src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumVariantViewOfEnumerator.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections;
+using System.Runtime.InteropServices.ComTypes;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    internal class EnumVariantViewOfEnumerator : IEnumVARIANT, ICustomAdapter, ICustomQueryInterface
+    {
+        private static readonly Guid s_IID_IManagedObject = new Guid("C3FCC19E-A970-11d2-8B5A-00A0C9B7C9C4");
```

Comment:
`IID_IManagedObject` was for  the magic promotion of COM to remoting. We do not do that in .NET Core. This special casing for `IID_IManagedObject` should not be needed here.

Reply:
In that case I'll remove it. 👍